src/Controller/SentencesController.php

```diff
@@ -828,6 +835,51 @@ public function edit_correctness()
         }
     }
 
+    /**
+     * Mark all sentences of a user as incorrect.
+     *
+     * @param string $username Username of the user. 
```

Comment:
You should change `username` to `id` here too.

Reply:
Oops, got it.